### PR TITLE
Testing parser

### PR DESCRIPTION
**Pull request by tvheems**
The pom file is updated and now maven runs with JaCoCo.
Some checkstyle fixes and cleanup after ParserTest.

### CODE REVIEW

File: src/main/java/datastructure/DrawNode.java [hunk 1]
```diff
@@ -3,7 +3,7 @@
 import javafx.scene.shape.Rectangle;
 
 /**
- * Created by Martijn on 18-5-2017.
```
Comment (jjtiessen) [inline src/main/java/datastructure/DrawNode.java]: Git blame Martijn.

**Comment by tvheems** [comment]
I just deleted it...

**Comment by jjtiessen** [comment]
OH very nice!

**Comment by tvheems** [comment]
# I TRY

**Comment by jjtiessen** [comment]
Imagine if we all would...

**Comment by jjtiessen** [comment]
Makes one of us...

**Pull request merged**